lib/exporter/exporter.go

```diff
@@ -191,14 +199,16 @@ func newMetricDescr(namespace string, metricName string, docString string, label
 }
 
 // NewRedisExporter returns a new exporter of Redis metrics.
-func NewRedisExporter(redisURI string, opts Options) (*Exporter, error) {
+func NewRedisExporter(redisURI string, opts Options, bi BuildInfo) (*Exporter, error) {
```

Comment:
Instead of changing the signature - what do you think about adding the BuildInfo fields to the Options struct?

Reply:
Yes, that's a much better idea. I'll do that.